Tests/LoggingTests/Test+Floats.swift

```diff
@@ -14,10 +14,10 @@ final class FloatTests: XCTestCase {
         logger.logLevel = .trace
 
         logger.debug("\(min, privacy: .public)")
-        XCTAssertEqual(logging.recorder.message, "\(CGFloat.leastNonzeroMagnitude)")
+        XCTAssertEqual(logging.recorder.message, String(format: "%.06f", CGFloat.leastNonzeroMagnitude))
```

Comment:
Changes to handle the floating point precision comparison issue (0.000000 != 5e-324 for CGFloat = 0 
> XCTAssertEqual failed: ("0.000000") is not equal to ("5e-324")

Since the code includes a default of 6 for precision it seemed appropriate to test against that same level of precision.